programs/server/Server.cpp

```diff
@@ -445,6 +455,39 @@ void checkForUsersNotInMainConfig(
     }
 }
 
+static void loadEncryptionKey(const std::string & key_command [[maybe_unused]], Poco::Logger * log)
+{
+#if USE_BASE64 && USE_SSL && USE_INTERNAL_SSL_LIBRARY
+
+    auto process = ShellCommand::execute(key_command);
```

Comment:
Another inconsistency - here the key is loaded by a command placed in config,
while for encrypted disks we don't allow to put a script in config file, we expect the key can be imported from env variable with `from_env=...` attribute in config.